M2/Macaulay2/tests/ComputationsBook/completeIntersections/test.out.expected

```diff
@@ -731,9 +731,9 @@ o108 : T'
 
 i109 : psi = map(K,B)
 
-o109 = map (K, B, {0, 0, 0})
+o109 = psi
 
-o109 : RingMap K <--- B
+o109 : RingMap K <--- B, given by {0, 0, 0}
```

Comment:
The idea of this output line is to simply give the type of the output.  The output line with an equal sign is where the output itself goes.

Reply:
Notice that the output two lines above has changed to just `psi`. That's because I changed `RingMap` to use global assignments instead.

Reply:
Either you want to see the value of psi in the output or you don't.  If you do, change RingMap back.

Reply:
No, this is how I wanted it, so that this would work nicely:
```m2
i1 : S = kk[x]; R = S[y];

i3 : f = map(R, S)

o3 = f

o3 : RingMap R <--- S, given by {x}

i4 : f_*

o4 = pushforward functor f_*

o4 : Functor
```
I want ring maps to have a global name so the pushforward functor is printed nicely while the information about the ring map is still visible in the `o3 : ` line.

Reply:
The output line with a colon on it is for the type only, not the value.  You can use `describe` to manually get more information about something that prints out as its name, as we do for rings.

Reply:
Okay, I'll remove the generators.